doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,54 @@
+.. role:: html(raw)
+   :format: html
+
+.. _embeddings:
+
+Quantum embeddings
+===================
+
+A *Quantum Embedding* represents classical data as quantum states in a Hilbert space, i.e., *a quantum feature map*. An embedding takes a classical datapoint :math:`x` and translates it into a set of gate parameters in a quantum circuit. This ends up creating a quantum state :math:`\mid \psi_x \rangle`. This process is a crucial part of designing quantum algorithms and affects their computational power. For more details, see :cite:`Mariabook2018` and :cite:`schuld2018quantum`. PennyLane provides built-in embedding templates that can do the embedding for the user; see :mod:`pennylane.templates.embeddings`. 
+
+
+Let's consider classical input data consisting of :math:`M` examples, with :math:`N` features each: 
+
+.. math:: \mathcal{D}=\{x^{1}, \ldots, x^{M}\}
```

Comment:
In this section, it might be better to use bracketed indices (x^{(M)}) or subscripts (x_m) to avoid it looking like exponentiation.